app/templates/components/edit-reminder.hbs

```diff
@@ -0,0 +1,3 @@
+{{#link-to 'reminders.reminder'}}
+  <button class='save-reminder-btn' {{action 'saveDirtyDiana' on='click'}}>Save</button>
```

Comment:
Side note - on click is a default button action so you don't need to specify that here.